### DISCUSSION TITLE
Regarding repository uploading

### DISCUSSION THREAD

**Discussion by AkarshRastogi20** [community/community#47153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Today I try to upload my repository on github but it shows me render unable to open rich display so what should it need to do . Even I also try by changing the dns settings but still got same message.

Anyone can help me please.

**Comment by KuganParthiban** [comment]
I am new to GitHub. I have been trying to upload a notebook and received error message 'Unable to render rich display'.
